.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+### Completed each of the following items before creating a PR
+
```

Comment:
You've said this twice... so I don't understand.... our guidelines say that a ticket number must be put in the PR description in addition to the commit messages:

(from: http://mooseframework.org/wiki/Contributing/ )
```
Further, every Pull Request also needs to reference an issue number in both its description. 
```